asserts/sysdb/trusted.go

```diff
@@ -130,7 +130,7 @@ func init() {
 // Trusted returns a copy of the current set of trusted assertions as used by Open.
 func Trusted() []asserts.Assertion {
 	trusted := []asserts.Assertion(nil)
-	if os.Getenv("SNAPPY_USE_STAGING_STORE") != "1" {
+	if !osutil.EnvBool("SNAPPY_USE_STAGING_STORE") {
```

Comment:
+1 for GetenvBool. Not a fan of Getenv either, but given upstreams' use the meaning becomes obvious at least.